### PR TITLE
P0 auth

### PR DESCRIPTION
**Pull request by dev-prabhat**
Implemented Authentication logic to Social Media App

Files to check(https://github.com/dev-prabhat/nerd-community/pull/2/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&show-viewed-files=true)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+         <Route path="/home" element={<Home/>}/>
+         <Route path="/explore" element={<Explore/>} />
+         <Route path="/bookmark" element={<Bookmark/>}/>
+         <Route path="/profile" element={<Profile/>}/>
+       </Route>
+
+       <Route element={<AuthRoute/>}>
+         <Route path="/" element={<Login/>}/>
+       </Route>
+     </Routes>
```
Comment (iRohitGaur) [inline src/App.js:21]: suggestion: make a 404 page

File: src/features/auth/authSlice.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+export const loginUser = createAsyncThunk("auth/login",async ()=>{
+    try {
+        const { data }  = await axios.post("/api/auth/login",{
+            username: "adarshbalika",
+            password: "adarshBalika123",
+        })
+        return data
+    } catch (error) {
+        console.log(error.response)
```
Comment (iRohitGaur) [inline src/features/auth/authSlice.js]: suggestion: remove `console.log` and handle error

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Gave some suggestions, please look into it.

Good to merge with updates 👍🏻

**Review by sudipkundu999 — APPROVED** [review]
The code looks good.
Do add a guest login feature.

**Pull request merged**